parsers/to-react-native/to-react-native.parser.ts

```diff
@@ -52,7 +52,6 @@ export default async function (
 
           if (!(<any>TokensClass)[tokenClassName]) return;
 
-          token.name = camelCase(token.name);
```

Comment:
To prevent breaking change, prefer adding an option `formatName` to the parser with camelCase by default.

```ts
export type OptionsType =
  | Partial<{
     //...
     formatName: 'camelCase' | 'kebabCase' | 'snakeCase' | 'pascalCase' | 'none';
     //...
    }>
  | undefined;
```
After that you can apply the right function and satisfy your case with the `none` option and prevent breaking change the default option to `camalCase`:
```ts
token.name = options?.formatName !== 'none' ? _.[options?.formatName ?? 'camelCase'](token.name) ? token.name;
```

Reply:
I understand that you want to prevent a breaking change, and will change it to a `formatName` option 😉  It's unfortunate however because a `formatName` option would be doing exactly the same as chaining a parser in front of this one.

Reply:
I'm also not a big fan of adding an extra option.

Do note that it is actually not really a breaking change per se. Since users of the `to-react-native` parser were essentially "forced" to use the `camelcasify` parser due to the name being explicitly camel cased in the `to-react-native` paser.

By removing this explicit camel casing of the name of a file, this won't change a thing for the user. Since now, the `to-react-native` parser effectively adheres to the transformed name of the file as it comes in (instead of altering it itself).

Reply:
I understand, but currently, some users use` to-react-native` without `camelCasify`. So these users will be impacted by this change. So we can't do a breaking change like that.

We are currently thinking of a versioning solution to satisfy these use cases.